code/UMLEditor.py

```diff
@@ -24,10 +24,20 @@
 
 ##########################################################################
 
-# **Write Documentation Here**
-def prompt_exit():
-    print ("To be implemented")
+# Exits out of UMLEditor
+def prompt_exit(model:UMLModel):
+    print ("Do you want to save current working project?")
+    userInput = input().upper()
+
+    # if user wants to save project, call the save_model function to save working project
+    if userInput == 'YES':
+        model.save_model()
+
+    if userInput == 'NO':
+        print("Goodbye!")
 
+    exit()
```

Comment:
If the user types something other than yes or no, then it will exit without saving. We might want to force the user to pick an option so they dont accidentally exit (like quickly typing yes and mistyping it). You could probably put the prompt in a loop, and loop until it gets a valid answer. 

Also, I believe it might be a good idea to give the user the option to cancel, if they decide that they don't want to exit.